cdap-docs/reference-manual/source/cli-api.rst

```diff
@@ -399,4 +399,3 @@ These are the available commands:
    ``remove role <role-name> from <principal-type> <principal-name>``,"Removes a role from a principal in authorization system for role based access control."
    ``revoke actions <actions> on entity <entity-id> from <principal-type> <principal-name>``,"Revokes a user's permission to perform certain actions on an entity. <actions> is a comma-separated list."
    ``revoke all on entity <entity-id>``,"Revokes all privileges for all users on the entity."
-   ``security access entity <entity-id> principal-type <principal-type> principal-name <principal-name> action <action>``,"Checks whether a principal is authorized to perform an action on an entity."
```

Comment:
Yes. It was in `/cdap_3.3/cdap-cli/src/main/java/co/cask/cdap/cli/command/security/CheckActionCommand.java`, but that file has been removed as 3.4. Looks like @bdmogal deleted it in this PR: https://github.com/caskdata/cdap/pull/5382. That PR has changed a number of items in an undocumented fashion.

I have filed https://issues.cask.co/browse/CDAP-5606 to have this addressed, and have put this PR on hold until this is resolved, as this might need to be revised depending on how this is handled.